src/utils/constants.ts

```diff
@@ -19,3 +19,9 @@ export const REPORTS_FAILED_PAYMENTS_DIRECTORY = join(
   "payments",
   "failed"
 );
+
+export const CUSTOM_DELEGATOR_REPORT_DIRECTORY = join(
+  REPORTS_DIRECTORY,
+  "custom",
```

Comment:
Why `custom` 🤔

Reply:
Removed custom and just left "delegator"